parteh/PRTAllometricCarbonMod.F90

```diff
@@ -456,7 +459,11 @@ subroutine DailyPRTAllometricCarbon(this)
     call bdead_allom( target_agw_c, target_bgw_c, target_sapw_c, ipft, target_struct_c)
     
     ! Target leaf biomass according to allometry and trimming
-    call bleaf(dbh,ipft,canopy_trim,target_leaf_c)
+    if(leaf_status==2) then
```

Comment:
maybe this is for another PR, but it would be nice to give names to these integer phenology flags in https://github.com/NGEET/fates/blob/master/main/FatesConstantsMod.F90 and use them throughout the code.  Whenever I look at phenology code I always find myself having to look up what the integers mean.

Reply:
Hmmm, this is the label we would want instead of "2":
https://github.com/NGEET/fates/blob/master/main/EDTypesMod.F90#L80

The only problem is that this parteh module, with the interest of modularity in mind, does not access EDTypesMod.  It does access FatesConstantsMod.F90, perhaps we could start putting a lot of these parameter constants (labels) in that file...  I actually like that a lot.

Here is the top of the module where we list the usable modules:
https://github.com/NGEET/fates/blob/master/parteh/PRTAllometricCarbonMod.F90#L14

Reply:
I could create an issue suggesting we migrate parameter constants in EDTypesMod.F90 to FatesConstantsMod.  @ckoven what say you?

Reply:
yes agreed.  for some github reason that I don't understand my comments above were marked as pending, but, yes it seems like we both arrived at the same conclusion independently.